modules/cnv-configuring-live-migration-limits.adoc

```diff
@@ -0,0 +1,40 @@
+
+// Module included in the following assemblies:
+//
+// cnv_users_guide/cnv-live-migration-limits.adoc
+
+[id="cnv-configuring-live-migration-limits_{context}"]
+= Configuring live migration limits
```

Comment:
I'm not sure about the word `limits`. There can be different options, not just limiting factors. That's minor though...

Reply:
I've added "and timeouts" where appropriate.